image/store.go

```diff
@@ -118,8 +117,8 @@ func (is *store) restore() error {
 }
 
 func (is *store) Create(config []byte) (ID, error) {
-	var img Image
-	err := json.Unmarshal(config, &img)
+	var img *Image
+	img, err := NewFromJSON(config)
```

Comment:
So one difference is that `NewFromJSON` also propagates the `rawJSON` field;
https://github.com/moby/moby/blob/ca520bd1b807ace7c041bf416cb1bba385d968e5/image/image.go#L239

At a glance, I don't think it makes a difference for how it's used here (but thought I'd at least leave a comment)